_topic_maps/_topic_map.yml

```diff
@@ -1175,6 +1175,8 @@ Topics:
     File: using-pod-level-bonding
   - Name: Configuring hardware offloading
     File: configuring-hardware-offloading
+  - Name: Switching Bluefield-2 between NIC and DPU mode
```

Comment:
We want to switch _to_ DPU mode or _from_ DPU mode?

Reply:
no, we want to switch from DPU mode (which is the default) to NIC mode. We don't support switching back since we don't support DPU mode as of today.